addons/binding/org.openhab.binding.netatmo/README.md

```diff
@@ -106,28 +107,35 @@ Example item for the **indoor module**:
 Number Netatmo_Indoor_CO2 "CO2" <carbondioxide> { channel = "netatmo:NAMain:home:inside:Co2" }
 ```
 
-**Supported channels for the indoor module:**
-
-* Temperature
-* TemperatureTrend
-* Humidity
-* Co2
-* Pressure
-* PressureTrend
-* AbsolutePressure
-* Noise
-* HeatIndex
-* Humidex
-* Dewpoint
-* DewpointDepression
-* WifiStatus
-* Location
-* TimeStamp
-* LastStatusStore
-* MinTemp
-* MaxTemp
-* DateMinTemp
-* DateMaxTemp
+**Supported channels for the main indoor module:**
+
+| Channel ID          | Item Type | Description                                              |
+|---------------------|-----------|----------------------------------------------------------|
+| Co2                 | Number    | Air Quality in ppm                                       |
+| Temperature         | Number    | Current temperature                                      |
+| TempTrend           | String    | Temperature Evolution Trend (up, down, stable)           |
```

Comment:
lowercase `trend`?
For `PressTrend` we define/inform the user about a time range. Is it possible here, too?

Reply:
If this is not mentioned, I suppose we don't have this information.
@clinique : any idea ?

Reply:
@lolodomo : I confirm that the API does not give the information regarding the trend observation period